registration/manager_test.go

```diff
@@ -48,6 +49,7 @@ func TestNewServiceRegistrationManager(t *testing.T) {
 				TLSEnabled: false,
 				ServiceRegistration: &config.ServiceRegistrationConfig{
 					ServiceName: config.String("cts-service"),
+					Address:     config.String("172.1.2.3"),
```

Comment:
since this is shared between the service and the service registration, consider making this string a variable? Might make it slightly easier to change in the future if we need to without missing one of them.